testgrid/default-template.yaml

```diff
@@ -0,0 +1,38 @@
+# Default Settings are required for TestGrid YAML Configuration files
```

Comment:
What is this file for?

Reply:
It serves two purposes:
1) The specific use case I'm writing for, pulling annotations into a YAML file, still requires default settings. Both Configurator and the standalone solution would set those default settings by pulling them from a document like this.
2) Documentation. If there's going to be no single "config.yaml" with a bunch of comments at the top describing what all the defaults are, we should maintain a copy-pastable template that describes what all those defaults are.

If we insist on dashboards being defined in the YAML, then I would add to this template something like `# here's an example dashboard and all the dashboards you want need to go here`

Reply:
1. That makes sense and was my originaly assumption, but it doesn't seem to actually be referenced anywhere? Or am I just blind?

Reply:
I'm not pointing it out anywhere in this PR explicitly.

Updating the READMEs with how this and sharding change TestGrid workflows is a TODO in my near future. Is there something I could change to make this easier to discover?

Reply:
On further thought I think this is fine for now.